pkg/controller/operators/catalog/operator.go

```diff
@@ -1393,16 +1393,14 @@ func (o *Operator) syncResolvingNamespace(obj interface{}) error {
 	}
 
 	// Make sure that we no longer indicate unpacking progress
-	subs = o.setSubsCond(subs, v1alpha1.SubscriptionCondition{
-		Type:   v1alpha1.SubscriptionBundleUnpacking,
-		Status: corev1.ConditionFalse,
-	})
+	o.removeSubsCond(subs, v1alpha1.SubscriptionBundleUnpacking)
```

Comment:
If we want to keep the condition, _I think_ alternative fix will be something like this:

```suggestion
	o.setSubsCond(subs, v1alpha1.SubscriptionCondition{
		Type:   v1alpha1.SubscriptionBundleUnpacking,
		Status: corev1.ConditionFalse,
	})
```

Because `setSubsCond` returns a new slice of pointers containing updated subs only (similar to old implementation of `removeSubsCond`).

Reply:
Removing condition can potentially break clients, but given the fact:
1. It is relatively recent change
2. And not consistent with other conditions

I think it might be ok to remove it.

Reply:
My vote would be to remove it. Reasons I say that are: 
* When I asked around I wasn't able to get a "a user asked for it and hence it was added in there" answer.
* Which feeds into the doubts about the usefulness of clients needing to know if BundleUnpacking is true or false. As opposed to for example, the usefulness of clients needing to know if `ResolutionFailed` is true or false.

Reply:
Ship it! :shipit: